src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useCallback, useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
set loading at the start true, because on one milisecond at the begining renders table header, after it dissappear, and after that again renders either table or error